modules/serverless/pages/features/eventing-kafka-scaling/eventing-kafka-scaling-setup.adoc

```diff
@@ -0,0 +1,96 @@
+= Setup Autoscaling For Eventing Kafka Components
+:compat-mode!:
+// Metadata:
+:description: Setup Autoscaling for Eventing Kafka components in {serverlessproductname}
+
+This page describes how to set up the Autoscaling feature for Eventing Kafka, which allows
+for the Eventing Kafka components to autoscale based on the lag in the Kafka topic.
+Note: this is only currently for the KafkaSource.
+
+[IMPORTANT]
+====
+{serverlessproductname} autoscaling for Eventing Kafka components is a Developer Preview feature only.
+Developer Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
+Red Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Developer Preview features, see https://access.redhat.com/support/offerings/devpreview/.
+====
+
+== Prerequisites
+
+* You have access to an {product-title} account with clsuter administrator acccess.
+
+* Install the OpenShift CLI (`oc`).
+
+* Install the {serverlessoperatorname}.
+
+* Install the {custommetricsautoscaleroperatorname}.
+
+== Enable Autoscaling of KafkaSources in `EventingKafka`
+
+Enabling controller-autoscaling-keda in `KnativeKafka`
```

Comment:
```suggestion
Enabling `controller-autoscaling-keda` in `KnativeKafka`
```

Reply:
Let's also include a few sentences how what is the enabled behavior (things like autoscaling is applied by default to all KafkaSources with option to disabled it with `<link to disable autoscaling for a resource section>`)